src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,17 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
 
+    public static void main(String[] args) {
+        movieService.add(Movie.builder().title("Interstellar").description("My favorite movie")
+                .build());
+        System.out.println(movieService.get(1L));
```

Comment:
```suggestion
        System.out.println(movieService.get(interstellar.getId()));
```
In your implementation, if you run `movieService.add` method three times it will create three records in the movies table with different ids, but you always fetching the movie with id=1